src/python/dxpy/sugar/__init__.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2013-2019 DNAnexus, Inc.
+#
+# This file is part of dx-toolkit (DNAnexus platform client libraries).
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#   use this file except in compliance with the License. You may obtain a copy
+#   of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+from __future__ import print_function, unicode_literals, division, absolute_import
+from functools import wraps
+import re
+
+import dxpy
+
+from dxpy.sugar.processing import run_cmd, chain_cmds
+from dxpy.sugar.transfers import (
+    Uploader,
+    Downloader,
+    upload_file,
+    tar_and_upload_files,
+    download_file,
+)
+
+
+MEM_RE = re.compile(r"^MemTotal:[\s]*([0-9]*) kB")
+MEM_CONVERSIONS = {
```

Comment:
TIL that `/proc/meminfo` stores memory in base 2 not base 10. The `kB` output misled me.

That being said, do we want to to use the same suffixes outputted when you read the file (`kB`). Or even do you want to coerce to `KiB`.

I'm still of the understanding that `KB` implies base-10. Either way the change would be minor and only impact readability of the code. Thoughts on the correct approach here?

Reply:
In the docstring under Returns I use  KiB, MiB, GiB. I'm not sure where else I'd use them here. But yes, we should use those same suffixes when printing the memory to the log.